src/components/overview/sub-Components/ImageModal.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const ImageModal = ({ title, children, onExitClick }) => {
```

Comment:
This seems similar to the generic modal we created. Could that have been reused?